manticore/core/plugin.py

```diff
@@ -10,20 +10,32 @@
 logger = logging.getLogger(__name__)
 
 
-class Plugin:
+class DecorateAllMeta(type):
+    @staticmethod
+    def _if_enabled(f):
```

Comment:
You've added `_if_enabled` here, but not removed it from the subclass. While I think it's technically fine, the code duplication will get confusing.